### PR TITLE
[IMP] mail: consistent default visual of thread topbar icon buttons

### PR DESCRIPTION
**Pull request by alexkuhn**
- Consistent icon color by default (dark grey)
- Hover feedback visual for all buttons
- Rounded border on hover

Task-2860749

### CODE REVIEW

File: addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml [hunk 1]
```diff
@@ -50,30 +50,30 @@
...
+                            <i class="fa fa-lg fa-video-camera text-700"/>
                         </button>
                     </t>
                     <t t-if="threadViewTopbar.thread and threadViewTopbar.thread.hasInviteFeature">
-                        <button class="o_ThreadViewTopbar_inviteButton o_ThreadViewTopbar_button btn px-2 border-none bg-transparent shadow-none"  t-attf-class="{{ threadViewTopbar.invitePopoverView ? 'opacity-100 opacity-75-hover o-active' : 'opacity-75 opacity-100-hover' }}" title="Add users" t-on-click="threadViewTopbar.onClickInviteButton" t-ref="inviteButton">
-                            <i class="fa fa-lg fa-user-plus" t-attf-class="{{ threadViewTopbar.invitePopoverView ? 'text-800' : 'text-600' }}"/>
+                        <button class="o_ThreadViewTopbar_inviteButton o_ThreadViewTopbar_button btn px-2 border-none rounded shadow-none opacity-100"  t-attf-class="{{ threadViewTopbar.invitePopoverView ? '' : '' }}" title="Add users" t-on-click="threadViewTopbar.onClickInviteButton" t-ref="inviteButton">
```
Comment (seb-odoo) [inline addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml:61]: If opacity is always 100 I guess the value is now useless?

Comment (alexkuhn) [reply addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml]: Changes done (removed `opacity-100` for all of them that had no `opacity-hover`)

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Comment by alexkuhn** [comment]
Just rebased

**Comment by seb-odoo** [comment]
@robodoo r+

**Pull request closed without merging**